akka-http-core/src/main/scala/akka/http/impl/engine/ws/FrameHandler.scala

```diff
@@ -24,10 +24,17 @@ import scala.util.control.NonFatal
 @InternalApi
 private[http] object FrameHandler {
 
+  private val PingFullFrame: FrameStart = FrameEvent.fullFrame(Opcode.Ping, None, ByteString.empty, fin = true)
+  final val mkDirectAnswerPing = () ⇒ DirectAnswer(PingFullFrame)
+
+  private val PongFullFrame: FrameStart = FrameEvent.fullFrame(Opcode.Pong, None, ByteString.empty, fin = true)
+  final val mkDirectAnswerPong = () ⇒ DirectAnswer(PongFullFrame)
+
```

Comment:
Shouldn't these rather go in the `WebSocket` companion as that is where they are used?